app/resonant-laboratory/models/Dataset.js

```diff
@@ -1,3 +1,4 @@
+import d3 from 'd3';
```

Comment:
Yeah, that's D3 v4... It will take a lot of revision across the app, so we tucked it all in #280. For now, I'm sticking with D3 v3.